nautobot/extras/tests/test_filters.py

```diff
@@ -954,6 +956,52 @@ def test_search(self):
         )
 
 
+class LogEntryTestCase(FilterTestCases.FilterTestCase):
+    queryset = LogEntry.objects.all()
+    filterset = LogEntryFilterSet
+
+    def setUp(self):
+        super().setUp()
+        # Only staff can see the logs.
+        self.user.is_staff = True
+        self.user.save()
+
+        admin_user = User.objects.create(username="admin", is_staff=True)
+        content_type = ContentType.objects.get_for_model(User)
+
+        self.log_entries = []
+        for action_flag, action_label in ACTION_FLAG_CHOICES:
+            entry = LogEntry.objects.log_action(
+                user_id=admin_user.pk,
+                content_type_id=content_type.pk,
+                object_id=self.user.pk,
+                object_repr=str(self.user),
+                action_flag=action_flag,
+                change_message=f"{action_label} {str(self.user)}",
+            )
+            self.log_entries.append(entry)
+
+    def test_action_flag(self):
+        params = {"action_flag": ["Deletion"]}
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset(params, self.queryset).qs, self.queryset.filter(action_flag=3)
+        )
+
+    def test_search(self):
+        params = {"q": "admin"}
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset(params, self.queryset).qs, self.queryset.filter(user__username__icontains="admin")
+        )
+        params = {"q": "nautobotuser"}
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset(params, self.queryset).qs, self.queryset.filter(object_repr__icontains="nautobotuser")
+        )
```

Comment:
These two are going to match **all** LogEntry instances created in `setUp` above, won't they? Not ideal for a filtering test case since it's identical to fail-open behavior for the filterset.